Calibration/PPSAlCaRecoProducer/test/test_prompt_PPSAlCaReco_output.py

```diff
@@ -0,0 +1,81 @@
+import FWCore.ParameterSet.Config as cms
+
+# load era modifier to run on 2018 data
+from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
+
+process = cms.Process( 'TEST',ctpps_2018)
+
+# LHCInfo plotter
+process.load("Validation.CTPPS.ctppsLHCInfoPlotter_cfi")
+process.ctppsLHCInfoPlotter.outputFile = "alcareco_lhc_info_prompt.root"
+
+# Load geometry from DB
+process.load("Geometry.VeryForwardGeometry.geometryRPFromDB_cfi")
+
+# track distribution plotter
+process.ctppsTrackDistributionPlotter = cms.EDAnalyzer("CTPPSTrackDistributionPlotter",
+  tagTracks = cms.InputTag("ctppsLocalTrackLiteProducerAlCaRecoProducer"),
+
+  rpId_45_F = cms.uint32(23),
+  rpId_45_N = cms.uint32(3),
+  rpId_56_N = cms.uint32(103),
+  rpId_56_F = cms.uint32(123),
+
+  outputFile = cms.string("alcareco_tracks_prompt.root")
+)
+
+# reconstruction plotter
+process.ctppsProtonReconstructionPlotter = cms.EDAnalyzer("CTPPSProtonReconstructionPlotter",
+  tagTracks = cms.InputTag("ctppsLocalTrackLiteProducerAlCaRecoProducer"),
+  tagRecoProtonsSingleRP = cms.InputTag("ctppsProtonsAlCaRecoProducer", "singleRP"),
+  tagRecoProtonsMultiRP = cms.InputTag("ctppsProtonsAlCaRecoProducer", "multiRP"),
+
+  rpId_45_F = cms.uint32(23),
+  rpId_45_N = cms.uint32(3),
+  rpId_56_N = cms.uint32(103),
+  rpId_56_F = cms.uint32(123),
+
+  outputFile = cms.string("alcareco_protons_prompt.root")
+)
+
+import HLTrigger.HLTcore.hltEventAnalyzerAOD_cfi
+process.hltAOD = HLTrigger.HLTcore.hltEventAnalyzerAOD_cfi.hltEventAnalyzerAOD.clone()
+process.hltAOD.processName = cms.string("HLTX")
+process.hltAOD.triggerResults = cms.InputTag("TriggerResults","","HLTX")
+process.hltAOD.triggerEvent   = cms.InputTag("hltTriggerSummaryAOD","","HLTX")
+process.hltAOD.triggerName = cms.string("HLT_PPSMaxTracksPerRP4_v1")
+process.hltAOD.stageL1Trigger = cms.uint32(2)
```

Comment:
minor comment: this makes the output of the [unit test somewhat verbose](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-92e153/22433/unitTests/src/Calibration/PPSAlCaRecoProducer/test/testPromptPPSAlCaRecoOutput/testing.log): given the urgency perhaps can be followed up later.

Reply:
Good point - it didn't look that bad in the terminal... :) We'll get back to it in the next PR.